<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>986</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="sendStop">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPortNumber">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM7</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditReport">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>1001</width>
     <height>851</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBaud">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushClear">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>911</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSend">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send One </string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendFileButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>80</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushEcho">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSave">
   <property name="geometry">
    <rect>
     <x>1144</x>
     <y>80</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendBinary">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Binary</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
